### PR TITLE
French translation complete

### PR DESCRIPTION
**Pull request by clueware**
I was interested in learning more about Hoodie, and given that the current attempt at a french translation was a bit old, the best way to learn was to translate the original documentation in my native language.

I've just completed my last round of spelling and grammatical checks, it should be OK now. If it suits the community, I'll start hunting for french alternatives to all external URLs - as long as it's not base pages like node.js website.

Comments and corrections would be greatly appreciated.

### CODE REVIEW

**Comment by gr2m** [comment]
:heart_eyes_cat: wow thanks so much for all the efforts!! I don’t speak French myself so I can’t properly review, but we really appreciate you took the time for all the translation work! Welcome to Hoodie :)

**Comment by clueware** [comment]
As long as I was going to go through it, I had nothing to lose to translate it on the fly. I'll try to update it in sync with the commits here so you can find someone to review it properly.

**Comment by clueware** [comment]
Just rebased on your latest modifications.

**Comment by gr2m** [comment]
Thanks! We reached out to the community to get a review from a another French speaking person: https://twitter.com/hoodiehq/status/724651213871034368

I hope we can merge this in soon :)

File: fr/community/contributor-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Dans ce document vous découvrirez notre processus d'embarquement pour les nouvelles personnes qui veulent nous rejoindre pour travailler sur Hoodie.
+
+Ce document est destiné aux membres de la communauté. Il vise à être un guide pour les étapes importantes à suivre et un rappel de comment collaborer avec cette communauté.
+
+Selon notre [Code de conduite](http://hood.ie/code-of-conduct.html), notre objectif est de fournir un espace sûr pour tous dans cette communauté et nous accueillons avec joie tout le monde, y compris les développeurs et personnes qui souhaitent devenir des contributeurs hors code.
+
+Ce document est et sera toujours en chantier. Nous essayons constamment d'améliorer notre engagement et notre culture communautaire et sommes toujours heureux d'avoir des retours!
+
+###  Qu'est-ce que les gens contribuent et où?
```
Comment (lucaspinto) [inline fr/community/contributor-onboarding.md]: Le point interrogation et d'exclamation est généralement précédé d'une espace fine insécable : http://unicode.org/udhr/n/notes_fra.html

File: fr/community/contributor-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Hoodie est un projet plutôt grand et nous avons plusieurs parties qui ont besoin d'être créées et / ou maintenues. Pour en savoir plus où nous avons besoin d'aide, jetez un oeil sur [le site de contribution](http://hood.ie/contribute/). Si vous ne trouvez rien que vous ayez envie de faire ou avez des questions additionnelles, merci de [nous contacter](http://hood.ie/contact/).
+
+---------------
+
+### 1. Dire bonjour!
+
+- En tant que premier contact pour un nouveau contributeur, vous êtes responsable de les intégrer à la communauté.
+- si ça ne semble pas pertinent, ou si ça n'est pas possible pour quelque raison que ce soir, par exemple vos propres obligations, manque de temps ou raisons personnelles, merci d'en parler à la communauté le plus tôt possible pour voir qui peut être leur contact et les guider dans les semaines et mois à venir.
```
Comment (lucaspinto) [inline fr/community/contributor-onboarding.md]: "quelque raison que ce soi_t_"

File: fr/community/contributor-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- Merci de garder à l'esprit que, idéalement, deux personnes qui partagent le même domaine, les mêmes intérêts ou compétences peuvent apprendre l'un de l'autre. 
+- Une fois que vous avez trouvé quelqu'un, introduisez-les l'un à l'autre.
+
+Essayer de parler au nouveau contributeur. Nous avons appris que ceci permet de lever de nombreux obstacles à la communication et que ça permet aux choses d'avancer plus vite. De plus, rencontrer les gens est toujours amusant!
+
+La manière la plus facile de faire ça est un rapide <a href="http://talky.io" target="_blank">Talky</a>, un appel Skype ou Google Hangout entre la personne intéressée et au moins un contributeur (à commencer par vous). Si vous êtes dans la même ville, aller déjeuner, prendre un café ou une balade est toujours une option préférable.
+
+### 2. Commencer ensembles
```
Comment (lucaspinto) [inline fr/community/contributor-onboarding.md]: Ensemble ne prend pas de s.

File: fr/community/contributor-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+---------------
+
+### 1. Dire bonjour!
+
+- En tant que premier contact pour un nouveau contributeur, vous êtes responsable de les intégrer à la communauté.
+- si ça ne semble pas pertinent, ou si ça n'est pas possible pour quelque raison que ce soir, par exemple vos propres obligations, manque de temps ou raisons personnelles, merci d'en parler à la communauté le plus tôt possible pour voir qui peut être leur contact et les guider dans les semaines et mois à venir.
+- Merci de garder à l'esprit que, idéalement, deux personnes qui partagent le même domaine, les mêmes intérêts ou compétences peuvent apprendre l'un de l'autre. 
+- Une fois que vous avez trouvé quelqu'un, introduisez-les l'un à l'autre.
+
+Essayer de parler au nouveau contributeur. Nous avons appris que ceci permet de lever de nombreux obstacles à la communication et que ça permet aux choses d'avancer plus vite. De plus, rencontrer les gens est toujours amusant!
```
Comment (lucaspinto) [inline fr/community/contributor-onboarding.md]: "Essayez" doit prendre un z pour respecter la cohérance avec le paragraphe précédent.

File: fr/community/contributor-onboarding.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+La manière la plus facile de faire ça est un rapide <a href="http://talky.io" target="_blank">Talky</a>, un appel Skype ou Google Hangout entre la personne intéressée et au moins un contributeur (à commencer par vous). Si vous êtes dans la même ville, aller déjeuner, prendre un café ou une balade est toujours une option préférable.
+
+### 2. Commencer ensembles
+
+1. Présentez-leur notre [Code de Conduite](http://hood.ie/code-of-conduct/). Ne faites pas que le mentionner mais demandez-leur de le lire et demande-leur s'ils ont des questions et inquiétude. Si c'est le cas, répondez-y.
```
Comment (lucaspinto) [inline fr/community/contributor-onboarding.md]: Il manque un "z" au deuxième "demandez-leur".

**Comment by lucaspinto** [comment]
Hey @clueware, would you prefer comments or a commit on top of yours?
Edit: I feel weird pointing out typos when I could correct them myself.

File: _config.yml [hunk 6]
```diff
@@ -178,3 +181,61 @@ locales:
...
+      en: "anglais"
+      de: "allemand"
+      ru: "russe"
+      fr: "français"
+    nav:
+      hoodieverse: "Hoodieverse"
+      start: "Démarrage"
+      tutorials: "Tutoriels"
+      api: "API"
+      plugins: "Plugins"
```
Comment (nono) [inline _config.yml:197]: On traduit parfois plugins par greffons

File: fr/community/contributor-onboarding.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
+---
+layout: layout-community
+locales: fr
+---
+
+## A propos de ce document
```
Comment (nono) [inline fr/community/contributor-onboarding.md]: Vu qu'il y a un accent sur Être ci-dessous, il faudrait aussi mettre À ici

File: fr/community/irc-office-hours.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
+---
+layout: layout-community
+locales: fr
+---
+# Introduction
+
+Ce document est dédié aux Office Hours pour les membres et contributeurs de la communauté Hoodie qui se sont engagés comme responsables de l'IRC de Hoodie. Ces Office Hours sont aussi documents sur le [site web Hoodie](http://hood.ie/irc).
```
Comment (nono) [inline fr/community/irc-office-hours.md]: documents -> documentés

File: fr/community/irc-office-hours.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+nous sommes heureux que vous assuriez les Office Hours IRC de Hoodie aujourd'hui!
+
+Afin de rendre le tout aussi facile que possible pour vous, nous allons vous demander de suivre ce petit guide:
+
+### 1. Surveillez le temps
+
+Les Office Hours IRC de Hoodie ont lieux quotidiennement les jours de semaines de 14 à 18h UTC+2. En cas de doute sur les fuseaux horaires, allez voir <a href="http://everytimezone.com/" target="_blank">cette page</a>.
+
+Merci de garder à l'esprit que tout ceux sur le canal ne sont pas sur le même fuseau horaire, aussi il est utile de mentionner le fuseau concernant quand on parle d'hoarires ou de dates. :)
```
Comment (nono) [inline fr/community/irc-office-hours.md]: hoarires -> horaires

File: _config.yml [hunk 10]
```diff
@@ -178,3 +181,61 @@ locales:
...
+      works: "Comment Hoodie fonctionne"
+      glossary: "Glossaire"
+      next: "Démarrage"
+    menustart:
+      installation: "Installation"
+      started : "Démarrer avec Hoodie - partie 1"
+      next: "Tutoriels"
+    menututorials:
+      started : "Démarrer avec Hoodie - partie 2"
+      timetracker: "Gestion de temps"
```
Comment (almet) [inline _config.yml]: Gestion du temps ?

File: fr/community/irc-office-hours.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+### 2. Démarrage - IRC et Twitter
+
+Quand il est temps de démarrer les Office Hours (merci de démarrer à l'heure), nous vous demanderons de:
+
+#### 2a) dire bonjour sur IRC
+
+Envoyez un court message "Bonjour à tous, je suis présent" sur IRC. Voici quelques exemples de message à utiliser ou ré-écrire:
+
+- Bonjour à tous, les Office Hours IRC commencent! Si vous avez quelques questions que ce soit, interpellez-moi. Je suis en ligne jusque 18h UTC+2.
```
Comment (nono) [inline fr/community/irc-office-hours.md]: quelque question (au singulier)

File: fr/community/contributor-onboarding.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+
+ > "Être un committer ne veut pas dire que vous soumettez du code, ça veut dire que vous êtes engagé sur le projet."
+ > – <a href="https://community.apache.org/contributors/" target="_blank">Apache Software Foundation</a>
+
+Dans ce document vous découvrirez notre processus d'embarquement pour les nouvelles personnes qui veulent nous rejoindre pour travailler sur Hoodie.
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: j'utiliserais plutôt "acceuil" ici.

File: fr/community/irc-office-hours.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+- Salut les gens, content de vous voir! Je suis ici pour les Office Hours Hoodie à partir de maintenant et jusque 18h UTC+2. Merci de me pinger si vous avez quelques question, retour ou GIFs que vous souhaiteriez porter à notre connaissance.
+- Salut à tous, nous sommes de retour pour les Office Hours! Je suis ici pour vous aider, s'il y a quoique ce soit que je puisse faire pour vous: je suis connecté pour les 4 prochaines heures à partir de maintenant.
+- … [votre idée ici]
+
+Oh, oui, vous pouvez évidemment ajouter un GIF (pas forcément [celui-là](http://www.tehcute.com/pics/201204/bunny-falls-asleep-at-desk.jpg)) à tout moment... (il s'avère que la communauté Hoodie adore les GIFs).
+
+####2b) envoyez un Tweet
+
+Envoyer un court Tweet depuis le compte @hoodiehq, ou, si vous préférez, depuis votre compte personnel Twitter. Si vous n'avez pas accès à @hoodiehq, tweetez juste depuis votre compte pour qu'il puisse être retweeté depuis @hoodiehq) (ouais, nous sommes aussi rapide que ça). Votre tweet pourrait être quelque-chose comme …
```
Comment (nono) [inline fr/community/irc-office-hours.md]: rapides (avec un s à la fin)

File: fr/community/contributor-onboarding.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+Ce document est destiné aux membres de la communauté. Il vise à être un guide pour les étapes importantes à suivre et un rappel de comment collaborer avec cette communauté.
+
+Selon notre [Code de conduite](http://hood.ie/code-of-conduct.html), notre objectif est de fournir un espace sûr pour tous dans cette communauté et nous accueillons avec joie tout le monde, y compris les développeurs et personnes qui souhaitent devenir des contributeurs hors code.
+
+Ce document est et sera toujours en chantier. Nous essayons constamment d'améliorer notre engagement et notre culture communautaire et sommes toujours heureux d'avoir des retours!
```
Comment (almet) [inline fr/community/contributor-onboarding.md:17]: Espace avant le "!"

File: fr/community/irc-office-hours.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+
+###3. Allez-y!
+
+- Souvenez-vous du [Code de Conduite](http://hood.ie/code-of-conduct.html) et respectez-le *scrupuleusement*.
+- Soyez amical et patient.
+- Si vous avez besoin de plus de 2 minutes pour répondre à une question, faites savoir à votre correspondant que vous êtes toujours là et que vous travaillez le sujet
+- Si vous ne pouvez pas répondre à une question par vous-même, merci de:
+  - vérifier la [FAQ](http://faq.hood.ie)
+  - demander à la personne ayant posé la question de créer un ticket dans notre [dépôt "discussions"]https://github.com/hoodiehq/discussion/issues/new) ou dans la [FAQ](https://github.com/hoodiehq/faq/issues/new)
+- Si vous avez besoin de quitter le canal ou fare une pause pour quelque raison que ce soit, merci de vous assurer que quelqu'un peu prendre le relais.
```
Comment (nono) [inline fr/community/irc-office-hours.md]: fare -> faire
peu -> peut

File: fr/community/irc-office-hours.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+Essayer d'éviter de demander à d'autres membres de la communauté de rejoindre les conversations, à moins qu'il s'agisse d'un cas réellement urgent ou qu'ils aient évoqué la possibilité auparavant. Vous pouvez gérer ça, et nous essaierons tout pour régler votre problème au plus vite.
+
+###4. Souvenez-vous de notre FAQ
+
+Si vous recevez une question intéressante de la part de quelqu'un qui pourrait intéresser un plus grand nombre de personnes, merci de [la poster sur notre FAQ](https://github.com/hoodiehq/faq/issues/new), idéalement en incluant la solution, ou copiez-collez là quelquepart pou l'ajouter plus tard. Ceci permet d'aider pour les questions récurrentes et c'est une bonne façon de rendre du contenu accessible à tous tout en réalisant du support utilisateur de manière soutenable.
```
Comment (nono) [inline fr/community/irc-office-hours.md]: pou -> pour

File: fr/community/irc-office-hours.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+
+###5. Dites au revoir sur IRC
+
+Quand les Office Hours sont terminées, dites au revoir sur IRC et laisser un message où les gens pourrons nous joindre (c'est aussi indique sur le message de bienvenue du canal IRC, mais c'est toujours bon de montrer que nous y pensons). Ca pourrait être par exemple:
```
Comment (nono) [inline fr/community/irc-office-hours.md]: pourrons -> pourront
indique -> indiqué

File: fr/community/irc-office-hours.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+
+- Les Office Hours sont terminées pour aujourd'hui. Merci à tous d'avoir été avec nous aujourd'hui, et passez une bonne soirée. Si vous avez des question ou des retours à faire, visitez notre page http://hood.ie/irc où vous trouverez tous les endroits où ouvrir des tickets. Nous nous en occuperons aussi vis que possible. Merci!
```
Comment (nono) [inline fr/community/irc-office-hours.md]: vis -> vite

File: fr/community/contributor-onboarding.md [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+
+###  Qu'est-ce que les gens contribuent et où?
+
+Hoodie est un projet plutôt grand et nous avons plusieurs parties qui ont besoin d'être créées et / ou maintenues. Pour en savoir plus où nous avons besoin d'aide, jetez un oeil sur [le site de contribution](http://hood.ie/contribute/). Si vous ne trouvez rien que vous ayez envie de faire ou avez des questions additionnelles, merci de [nous contacter](http://hood.ie/contact/).
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: Remplacer "additionelles" par "supplémentaires"

File: fr/community/contributor-onboarding.md (hunk 5 above)
Comment (almet) [inline fr/community/contributor-onboarding.md]: inquiétude_s_

File: fr/community/contributor-onboarding.md [hunk 20]
```diff
@@ -0,0 +1,132 @@
...
+
+1. Présentez-leur notre [Code de Conduite](http://hood.ie/code-of-conduct/). Ne faites pas que le mentionner mais demandez-leur de le lire et demande-leur s'ils ont des questions et inquiétude. Si c'est le cas, répondez-y.
+2. Cernez leur situation:
+  - quelles compétences ont-il?
+     - sont-ils développeurs
+         - frontend / backend / mobile?
+     - font-ils autre-chose que du code?
+         - quel est leur spécialité?
+  - d'où vient leur intérêt dans Hoodie?
+  - sur quoi souhaitent-ils travailler?
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: ditto par rapport à la ponctuation.

File: fr/community/contributor-onboarding.md [hunk 21]
```diff
@@ -0,0 +1,132 @@
...
+  - quels compétences personnelles les motives?
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: les motiv_ent_

File: fr/community/contributor-onboarding.md [hunk 22]
```diff
@@ -0,0 +1,132 @@
...
+  - y a-t-il quelque-chose qu'ils veulent améliorer et / ou pour laquelle on peut les aider à se former?
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: remplacer "veulent" par "souhaitent"

File: fr/community/contributor-onboarding.md [hunk 23]
```diff
@@ -0,0 +1,132 @@
...
+3. Selon leurs compétences et connaissances, présentez-leur les termes et processus liés à Hoodie (voir [le glossaire Hoodie](../hoodieverse/glossary.html) et la [FAQ](http://faq.hood.ie/) pour plus d'information).
+
+
+### 3. Présentez-leur leur environnement
+
+Hoodie possède divers canaux de communication et processus qui peuvent être confondant pour les gens qui sont nouveaux dans la communauté. Durant cette période, votre travail est de vous assurer qu'ils savent comment communiquer efficacement avec le reste de la communauté et vous assurer qu'ils restent toujours informés.
```
Comment (almet) [inline fr/community/contributor-onboarding.md]: Remplacer "confondant" par "déconcertants" ? A default, confondant prends un s.

**Comment by clueware** [comment]
Thanks @lucaspinto, @nono and @almet for all for the corrections, I'll work on it right away - except for replacing "plugins" with "greffons": I one of those who consider that a technical term shouldn't be back-translated - a bit like "scaffolding". "Plugins" is understood by anyone working on software, "greffons" would just muddy understanding in my opinion.
I'll be back in a jiffy with the update!

File: fr/community/contributor-onboarding.md (hunk 22 above)
Comment (clueware) [inline fr/community/contributor-onboarding.md]: Je trouvais "souhaitent" moins catégorique que "veulent", mais c'est définitivement plus proche de "want" - for english speakers: @almet is right, it's "want", not "wish".

File: fr/community/contributor-onboarding.md (hunk 1 above)
Comment (clueware) [inline fr/community/contributor-onboarding.md]: Un typographe! The only hard part was to find the proper escape code: `&#x202F;`

File: fr/community/contributor-onboarding.md (hunk 20 above)
Comment (clueware) [inline fr/community/contributor-onboarding.md]: Deux typographes, bien ma veine! 😛

File: _config.yml (hunk 6 above)
Comment (clueware) [inline _config.yml:197]: @nono Je sais à propos de "greffons", mais comme dis plus bas je pars de ce que "plugins" ici est un mot technique, comme "scaffolding". La traduction n'apportera probablement rien à la compréhension, d'autant qu'il y a masse de référence à "plugin(s)" dans le code et les références `npm`. Je suis ouvert au débat, bien sûr. 😄

**Comment by clueware** [comment]
Incoming another round of fixes with all due respect to @lucaspinto ! Hang tight!

**Comment by lucaspinto** [comment]
:hand: thank you for your great work @clueware!

**Comment by clueware** [comment]
😄

**Pull request merged**